core/sr-primitives/src/generic/unchecked_mortal_extrinsic.rs

```diff
@@ -72,7 +73,8 @@ where
 	Call: Encode + Member,
 	Signature: Member + traits::Verify<Signer=AccountId>,
 	AccountId: Member + MaybeDisplay,
-	BlockNumber: SimpleArithmetic,
+	BlockNumber: SimpleArithmetic + AsPrimitive<u64>,
```

Comment:
Nitpick: `SimpleArithmetic` already requires that the type implements `AsPrimitive<u64.`, so this should be enforced implicitly.